kart/point_cloud/import_.py

```diff
@@ -291,7 +294,7 @@ def extract_pc_tile_metadata(pc_tile_path, extract_schema=False):
         {
             "type": "readers.las",
             "filename": str(pc_tile_path),
-            "count": 0,  # Don't read any individual points.
+            "count": 10,  # Don't read any individual points.
```

Comment:
magic number has a meaning?

Reply:
Oops nope - was testing what the stats filter did (ie does it read the extent from the header, or calculate an extent from the points read). This should still be zero. Reverted.